### DISCUSSION TITLE
discord intregration plugin connection failed

### DISCUSSION THREAD

**Discussion by ahmrh** [pear-devs/pear-desktop#880] [Q&A]
somehow my discord activity doesn't show up, when i click reconnect it says connection failed (header) and connection closed (desc). Any idea how to fix that?

![image](https://user-images.githubusercontent.com/79359426/201502494-8587422a-3155-4cd8-9b65-34201b2483f9.png)

*edit now it shows this
![image](https://user-images.githubusercontent.com/79359426/201511461-c5713d04-b61e-48db-9e71-1a41a17a3c96.png)

**Comment by AkshayRane05** [comment]
same is happening with me. did u find any fix?

**Reply by ahmrh** [reply]
Not yet, it has been two days since i run package update (pacman -Syu) and since this happened. I thought it has something to do with that update so i asked here
